README.rst

```diff
@@ -0,0 +1,329 @@
+============================================
+Batch Data Processing in CDAP using Workflow
+============================================
+
+The Workflow system in `Cask Data Application Platform (CDAP) <http://cdap.io>`__
+allows specifying, executing, scheduling, and monitoring complex series of jobs
+and tasks. In this guide, you will learn how it can facilitate the execution of
+`MapReduce <http://research.google.com/archive/mapreduce.html>`__
+programs in parallel based on the evaluation of boolean predicates.
+
+What You Will Build
+===================
+
+This guide will take you through building a
+`CDAP application <http://docs.cdap.io/cdap/current/en/developers-manual/building-blocks/applications.html>`__
+that uses ingested raw purchase events (of the form '``<name> bought <n> <item>s for $<price>``') to compute
+the total purchases made by a user along with the total purchases made for a particular product.
```

Comment:
to compute |---| in parallel |---| the total purchases made by each user and the total purchases made for each product.